singularity/configure.py

```diff
@@ -25,18 +25,20 @@ def main():
     parser.add_argument('--celery-path', help='Path to the Celery bind path')
     parser.add_argument('--dpdash-path', help='Path to the DPdash bind path')
 
-    parser.add_argument('--rabbit-host', help='Rabbitmq host address')
     parser.add_argument('--mongo-host', help='MongoDB host address')
-    parser.add_argument('--rabbit-port', help='Rabbitmq port number', default=5971)
     parser.add_argument('--mongo-port', help='MongoDB port number', default=27018)
+    parser.add_argument('--rabbit-host', help='Rabbitmq host address')
+    parser.add_argument('--rabbit-name', help='Rabbitmq node name', default='rabbit2')
```

Comment:
Should we have the default to be just `rabbit`?

Reply:
Definitely. And I also think the default ports should be set back to what they were before you created this branch (`27017` for Mongo, etc).

That comes into play with my comment on the other PR: https://github.com/PREDICT-DPACC/dpdash/pull/11#issuecomment-824996683

I think these should have defaults that match the original defaults for each of these services, but then allow them all to be overridden in e.g. an `.env` file.

Reply:
But for now I've just moved all of the hard coded ports/names you set to the defaults for args in `configure.py`.

Reply:
Once we merge this PR into this repo's `multi-user-config` branch, I'll then start another PR for `multi-user-config` that uses an `.env` file for overriding these defaults. That way this branch is always "usable" from the standpoint of a multi-user setup.